content/architecture/access-control.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Access Control
+linktitle: Access Control
+aliases: [/rbac/]
+description: Managing Access Control
+date: 2017-02-01
```

Comment:
tbh I don't really know what all these tags mean & how they are used - but April 2019 surely, for this new content?

Reply:
it's the creation date, and this is a renamed file but the diff doesn't show it.